src/test/java/guitests/IssueSelectedEventTest.java

```diff
@@ -0,0 +1,56 @@
+package guitests;
+
+import javafx.scene.control.ContextMenu;
+import javafx.scene.control.MenuItem;
+import javafx.scene.input.KeyCode;
+import org.junit.Before;
+import org.junit.Test;
+import ui.UI;
+import ui.listpanel.ListPanel;
+import util.events.IssueSelectedEventHandler;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class IssueSelectedEventTest extends UITest{
+
+    private static final String PANEL_IDENTIFIER = "#dummy/dummy_col0";
+
+    @Test
+    public void noTriggerIssueSelectedOnRightClick_IssueInPanelRightClicked_IssueSelectedNotTriggered() {
```

Comment:
I think the convention is `feature_scenario_expected` So here it should be `eventTriggerOnIssueSelection_byRightClick_selectionEventNotTriggered` or something like that.